terraform/moj-cjse-bichard7.tf

```diff
@@ -91,6 +91,16 @@ module "moj-cjse-bichard7" {
       reason       = "Security Scanning"                  #  Why is this person being granted access?
       added_by     = "Ben Pirt <ben@madetech.com>"        #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
       review_after = "2021-12-31"                         #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
-    }
+    },
+    {
+      github_user  = "djr-made-tech"
+      permission   = "push"
+      name         = "David Roberts"                        #  The name of the person behind github_user
+      email        = "david.roberts@madetech.com" #  Their email address
+      org          = "Madetech"                                #  The organisation/entity they belong to
+      reason       = "CJSE Bichard Development"                  #  Why is this person being granted access?
+      added_by     = "Dom Tomkins <dom.tomkins@justice.gov.uk>"        #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
+      review_after = "2021-12-31"                         #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
+    }:
```

Comment:
Eek - @bjpirt you need to remove the `:`

Reply:
oops! not sure how that got in there!